tests/tests.yaml

```diff
@@ -270,21 +291,19 @@ simple_strict_recurrence:
       -
         - FinishedAt(2018-04-08)
       -
-        - Copied
-        - "PostponedStrictBy(Duration { secs: 172800, nanos: 0 })"
-      -
-        - RecurredFrom(2018-04-08)
+        - RecurredStrict
         - FinishedAt(2018-04-08)
+      -
+        - RecurredStrict
 
-# TODO: Unwanted behaviour
 recurrence_and_postponing:
   allowed_divergence: 50
   from:
-    - 2018-06-01 foo due:2018-06-20 rec:1m
+    - 2018-06-01 foo due:2018-06-20 rec:+1m
 
   to:
-    - x 2018-06-17 2018-06-01 foo due:2018-06-15 rec:1m
-    - 2018-06-17 foo due:2018-07-15 rec:1m
+    - x 2018-06-17 2018-06-01 foo due:2018-06-15 rec:+1m
+    - 2018-06-17 foo due:2018-07-15 rec:+1m
```

Comment:
Hmm… I think with this change we're missing a test case for non-strict recurrence.

For instance,
```
2018-06-01 foo due:2018-06-20 rec:1m
```
to
```
x 2018-07-04 2018-06-01 foo due:2018-06-20 rec:1m
2018-07-04 foo due:2018-08-04 rec:1m
```
(as just confirmed using `topydo` to manually complete the task)

Reply:
Thanks for the example !